Makefile

```diff
@@ -71,6 +71,7 @@ test: ## Run tests
 freeze-requirements: ## Pin all requirements including sub dependencies into requirements.txt
 	pip install --upgrade pip-tools
 	pip-compile requirements.in
+	pip3 install -r requirements.txt
 
```

Comment:
Hard to imagine using this dev fixture without wanting to install the new dependency in the local dev environment...